### DISCUSSION TITLE
Demo link on librechat.ai points to expired  librechat.cfd domain

### DISCUSSION THREAD

**Discussion by wireflo** [danny-avila/LibreChat#4805] [Troubleshooting] [answered]
The Demo link in the header points to an ad infested domain parking site - probably because the domain expired:
https://demo.librechat.cfd/
<img width="890" alt="image" src="https://github.com/user-attachments/assets/41421bb5-331d-4c31-acb6-1f2ee07b3681">

**Comment by fuegovic** [comment, marked as answer]
You can use this url to access our demo: https://chat.librechat.ai/

we'll update the link on the website soon

**Comment by logkirk** [comment]
Is there an ETA on this? After I clicked the demo link, the website asked me to turn off my content blockers in order to access the site. I did so, since I assumed it to be an official link associated with LibreChat. In my opinion, this is a security issue. Who knows what sort of trackers that site installs?

**Reply by berry-13** [reply]
updated

**Reply by logkirk** [reply]
Great, thank you!
